docs/csharp/whats-new/snippets/whats-new-csharp9/WeatherObservation.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NewCsharp9
+{
+    // <DeclareWeatherObservation>
+    public struct WeatherObservation
```

Comment:
Curious, could this be a `readonly struct` since it uses `init`?

Reply:
no.